example_scripts/run_supervised.py

```diff
@@ -1,14 +1,17 @@
-from joblib import dump, load
+from typing import Union
+
+from numpy import ndarray
+
 from bnl_ml.supervised.data import load_data, featurization
 from bnl_ml.supervised.training import train_all_models
 from pathlib import Path
 import numpy as np
 
 
-def featurize_and_normalize(X_train, X_test):
-    fe_train = featurization(X_train)
-    fe_test = featurization(X_test)
-    train_max = np.max(fe_train, axis=0)
+def featurize_and_normalize(x_train, x_test):
```

Comment:
I keep `X_train` and `X_test` as capitalized because they are matrices while `y_<>` is a vector. This is more aligned with many online examples, although it is a pep violation.